### DISCUSSION TITLE
How can I change "huggingface.co/immich-app" to "hf-mirror.com/immich-app"?

### DISCUSSION THREAD

**Discussion by nodis** [immich-app/immich#8875] [Q&A]
### The bug

I am unable to directly access the "huggingface.co/immich-app", which prevents me from pulling the CLIP model. However, I am able to access the mirror website "hf-mirror.com/immich-app". How can I change "huggingface.co/immich-app" to "hf-mirror.com/immich-app"?

### The OS that Immich Server is running on

Synology DSM 7.2

### Version of Immich Server

v1.101.0

### Version of Immich Mobile App

v1.101.0

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
.
```

### Your .env content

```Shell
.
```

### Reproduction steps

```bash
1.
2.
3.
...
```

**Comment by mertalev** [comment]
See [this](https://github.com/immich-app/immich/discussions/6417#discussioncomment-8153066) answer.
